src/tools/auth0/handlers/tenant.ts

```diff
@@ -66,19 +66,20 @@ export default class TenantHandler extends DefaultHandler {
     // Do nothing if not set
     if (!tenant) return;
 
-    const updatedTenant: Partial<Tenant> = {
-      ...tenant,
+    const updatedTenant: TenantSettingsUpdate = {
+      // TODO: What's wrong with the enums?
```

Comment:
Think it's this https://github.com/auth0/api2/blob/8925c49da64a117b19dbedb37e47e1365198a37c/api/tenants/schemas/tenant_settings_update_properties.js#L105